openapi3.yaml

```diff
@@ -201,4 +201,7 @@ components:
           type: array
           items:
             $ref: ./Schema/link.yaml#/components/schemas/Link
+        displayPath:
```

Comment:
why is it here instead of the mc-models?

Reply:
fixed